src/App.jsx

```diff
@@ -252,42 +253,42 @@ function App() {
               </div>
               {data
                 ? Object.keys(data).map((key) => (
-                    <div className="file" key={key}>
-                      <div
-                        className="full-Box"
-                        onClick={() => {
-                          setActiveFileId(key);
-                          if (!tabList.includes(key)) {
-                            setTabList([...tabList, key]);
-                          }
-                        }}
-                        onMouseEnter={() => {
-                          setUtils(key);
-                        }}
-                        onMouseLeave={() => {
-                          setUtils(null);
-                        }}
-                      >
-                        <div className={utils === key ? 'hover-name' : 'name'}>
-                          {data[key].name}
-                        </div>
-                        <div className={utils === key ? 'utils' : 'noUtils'}>
-                          <i
-                            className="bx bxs-edit utilities"
-                            style={{ color: '#abdafb' }}
-                            onClick={() => {
-                              renameFile(key);
-                            }}
-                          ></i>
-                          <i
-                            className="bx bx-trash"
-                            style={{ color: '#eb336c' }}
-                            onClick={handleDeleteFileClick(key)}
-                          ></i>
-                        </div>
+                  <div className="file" key={key}>
```

Comment:
This seems to be a Prettier-related change. I wonder what is going on with this. Maybe if we in introduce a `.prettierrc`, this issue might go away. That would the make VSCode Prettier plugin (which might be the source of this particular change) do the same thing as our `npm run prettier` script.